src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
```

Comment:
I would just call these `file` to avoid confusion, there's something else called `fileObject` inside the current session uploaded `files` that is used to retry uploads.

Reply:
Also change any other references from `fileObject` to `file`, not only here